downloader/src/main/kotlin/VcsHost.kt

```diff
@@ -237,13 +237,15 @@ private fun toGitPermalink(
     vcsUrl: URI, revision: String, path: String, startLine: Int, endLine: Int,
     startLineMarker: String, endLineMarker: String
 ): String {
+    fun String.isPathToMarkdownFile() = endsWith(".md") || endsWith(".markdown")
```

Comment:
How about setting `ignorecase = true` at least for the former?

Also, consider moving this function scope to file level as the scope still is reasonable small while the reader of the outer function is not forced to have a look at the implementation.

Reply:
> How about setting `ignorecase = true` at least for the former?

Actually, I believe it makes more sense for the latter, as I could imagine people writing e.g. `.MarkDown`, but I have never seen `.MD`. Done for both, anyway.

> Also, consider moving this function scope to file level

Originally, I didn't want to do this until the function is actually becoming useful elsewhere, but again done anyway.